### PR TITLE
feat: worked on task edit

### PR DESCRIPTION
**Pull request by Anishali2**

### CODE REVIEW

File: apps/web/lib/components/kanban-card.tsx [hunk 1]
```diff
@@ -154,9 +149,7 @@ export default function Item(props: ItemProps) {
 			alt: member.user.firstName
 		};
 	});
-	const { collaborativeSelect } = useCollaborative(memberInfo.memberUser);
 
-	const menu = <>{!collaborativeSelect && <UserTeamCardMenu memberInfo={memberInfo} edition={taskEdition} />}</>;
```
Comment (Anishali2) [inline apps/web/lib/components/kanban-card.tsx:159]: I've removed the menu from # 159 and added the whole component into line no # 180

**Comment by Anishali2** [comment]
I've fixed the working flow on menu clicks after that I'll add the remaining menu list which is described by @KostyaEver 
[Apex_1714686401652.webm](https://github.com/ever-co/ever-teams/assets/105014311/f943fd55-bc69-4d66-9148-789671c37ffd)

**Review by Cedric921 — COMMENTED** [review]
after edit Task,  Task Input should close when I click outside ( when it lost focus)

https://github.com/ever-co/ever-teams/assets/83704005/3aeb62c3-9f1d-432e-8803-ca3fbd05da01

**Comment by Anishali2** [comment]
> after edit Task, Task Input should close when I click outside ( when it lost focus)
> 
>  Screen.Recording.2024-05-03.at.9.35.17.AM.mov

Ok Got it

**Comment by evereq** [comment]
@Anishali2 please fix issue @Cedric921 found in separate PR

**Comment by Anishali2** [comment]
> @Anishali2 please fix issue @Cedric921 found in separate PR

Sure Sir

**Comment by evereq** [comment]
@Anishali2 also, create new branch based on Develop, don't use this branch anymore, thanks!

**Pull request merged**